tbump/test/test_main.py

```diff
@@ -30,9 +30,15 @@ def check_local_git_status(test_repo):
     assert "v1.2.41-alpha-2" in out
 
 
-def has_pushed(test_repo, pushed=True):
-    tbump.git.run_git(test_repo, "ls-remote", "origin", "refs/tags/v1.2.41-alpha-2")
+def hash_pushed_tag(test_repo):
```

Comment:
@dmerejkowsky `has` inadvertently became `hash`, also in `hash_pushed_branch`

Reply:
Too much crypto :)